### PR TITLE
EnemyFreeKickPlay isApplicable and invariantHolds tests

### PR DESCRIPTION
**Pull request by naraiamsakun**
### Description

  Created a blank world to test the play EnemyFreeKick. Implemented isApplicable and invariantHolds tests for the play.

### Testing Done

 unit test

### Resolved Issues

 resolves issue #2261

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [x ] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [ x] **Remove all commented out code**
- [ x] **Remove extra print statements**: for example, those just used for testing
- [x ] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue
- [x ] **Justify drops in code coverage**: If this PR results in a non-trivial drop in code coverage (a bot should post a coverage diagram as a comment), please justify why we can't test the code that's not covered.

### CODE REVIEW

**Review by jameschuang2002 — CHANGES_REQUESTED** [review]
I think you need to run the formatting script. I am also not sure whether you should change NORMAL_START to DIRECT_FREE_US to pass the gameplay test. I reach this conclusion after reading similar documents about isApplicable and invariantHolds tests.

File: src/shared/parameter/config_definitions/passing_config.yaml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (akhilveeraghanta) [inline src/shared/parameter/config_definitions/passing_config.yaml]: seems like a random change made it in here

File: src/software/ai/hl/stp/play/enemy_free_kick_play_test.cpp [hunk 2]
```diff
@@ -198,3 +199,27 @@ TEST_F(EnemyFreekickPlayTest, test_enemy_free_kick_chipper_robots_close_to_net)
...
+
+    //
+    world.updateGameState(
+        ::TestUtil::createGameState(RefereeCommand::DIRECT_FREE_THEM, RefereeCommand::DIRECT_FREE_THEM));
+
+    ASSERT_TRUE(enemy_free_kick_play.isApplicable(world));
+    ASSERT_TRUE(enemy_free_kick_play.invariantHolds(world));
+
+    world.updateGameState(
+        ::TestUtil::createGameState(RefereeCommand::NORMAL_START, RefereeCommand::DIRECT_FREE_THEM));
```
Comment (akhilveeraghanta) [inline src/software/ai/hl/stp/play/enemy_free_kick_play_test.cpp]: You could change this to be DIRECT_FREE_US to make sure we don't run this play on our free kick

File: src/software/ai/hl/stp/play/enemy_free_kick_play_test.cpp [hunk 3]
```diff
@@ -198,3 +199,27 @@ TEST_F(EnemyFreekickPlayTest, test_enemy_free_kick_chipper_robots_close_to_net)
...
+TEST(EnemyFreeKickPlayInvariantAndIsApplicableTest, test_invariant_and_is_applicable)
+{
+    auto play_config = std::make_shared<ThunderbotsConfig>()->getPlayConfig();
+
+    auto world = ::TestUtil::createBlankTestingWorld();
+
+    // EnemyFreeKickPlay: the play under test
+    auto enemy_free_kick_play = EnemyFreekickPlay(play_config);
+
+    //
```
Comment (akhilveeraghanta) [inline src/software/ai/hl/stp/play/enemy_free_kick_play_test.cpp]: Seems to be a stay comment here

**Review by akhilveeraghanta — CHANGES_REQUESTED** [review]
nice work! just a couple comments

**Review by jameschuang2002 — DISMISSED** [review]
looks good to me

File: src/software/ai/hl/stp/play/enemy_free_kick_play_test.cpp [hunk 4]
```diff
@@ -198,3 +199,27 @@ TEST_F(EnemyFreekickPlayTest, test_enemy_free_kick_chipper_robots_close_to_net)
...
+    // Ensure play will start and stay running for our free kick play
+    world.updateGameState(::TestUtil::createGameState(RefereeCommand::DIRECT_FREE_THEM,
+                                                      RefereeCommand::DIRECT_FREE_THEM));
+
+    ASSERT_TRUE(enemy_free_kick_play.isApplicable(world));
+    ASSERT_TRUE(enemy_free_kick_play.invariantHolds(world));
+
+    // Ensure doesn't run when NOT our free kick play
+    world.updateGameState(::TestUtil::createGameState(RefereeCommand::DIRECT_FREE_US,
+                                                      RefereeCommand::DIRECT_FREE_THEM));
```
Comment (jonathanlew) [inline src/software/ai/hl/stp/play/enemy_free_kick_play_test.cpp:221]: non-blocking: it'd be nice to test normal start as well

File: src/shared/parameter/config_definitions/passing_config.yaml (hunk 1 above)
Comment (jonathanlew) [reply src/shared/parameter/config_definitions/passing_config.yaml]: I still see a new line here

**Review by jonathanlew — COMMENTED** [review]
just a couple comments from me

File: src/proto/define_a_proto_to_simulate_standalone_simulator_config [hunk 5]
```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
```
Comment (jonathanlew) [inline src/proto/define_a_proto_to_simulate_standalone_simulator_config]: I think you forget to switch branches before working on your new ticket

**Review by jonathanlew — COMMENTED** [review]
LGTM except for the extra file

**Review by jonathanlew — APPROVED** [review]
LGTM!

**Review by akhilveeraghanta — APPROVED** [review]

**Pull request merged**